src/main/java/hudson/plugins/ec2/win/EC2WindowsLauncher.java

```diff
@@ -133,9 +128,10 @@ private WinConnection connectToWinRM(EC2Computer computer, PrintStream logger) t
                 WinConnection connection = new WinConnection(ip, computer.getNode().remoteAdmin, computer.getNode().getAdminPassword().getPlainText());
 
                 connection.setUseHTTPS(computer.getNode().isUseHTTPS());
+
                 if (!connection.ping()) {
-                    logger.println("Waiting for WinRM to come up. Sleeping 10s.");
-                    Thread.sleep(sleepBetweenAttemps);
+                    logger.println("Waiting for WinRM to come up. Sleeping " + DEFAULT_RETRY_DELAY_SECONDS + "s.");
+                    Thread.sleep(TimeUnit.SECONDS.toMillis(DEFAULT_RETRY_DELAY_SECONDS));
                     continue;
```

Comment:
Please use a single field (like before) for the ms value, I don't like the long line of duplication of the conversion code.